account-kit/signer/src/client/index.ts

```diff
@@ -350,7 +353,20 @@ export class AlchemySignerWebClient extends BaseSignerClient<ExportWalletParams>
   public override disconnect = async () => {
     this.user = undefined;
     this.iframeStamper.clear();
-    await this.iframeStamper.init();
+
+    // In the latest version of the TK iframe stamper, the
+    // IframeStamper instance seems to not be usable after
+    // clearing it, so we need to create a new instance.
+    const stamper = new IframeStamper({
+      iframeContainer: document.getElementById(
+        this.iframeConfig.iframeContainerId,
+      ),
+      iframeElementId: this.iframeConfig.iframeElementId,
+      iframeUrl: "https://auth.turnkey.com",
+    });
+    this.iframeStamper = stamper;
+    this.setStamper(stamper);
```

Comment:
shouldn't need this line actually b/c it already happens inside of `initSessionStamper()`
```suggestion
```